### PR TITLE
Add Member.pending

### PR DESCRIPTION
**Pull request by Thewsomeguy**
Added the `Member.pending` property as Discord has released the Rules Screening feature to all community servers.

### CODE REVIEW

**Comment by HeadTriXz** [comment]
Pending should be optional.

**Review by Dramex — DISMISSED** [review]

File: lib/gateway/Shard.js [hunk 1]
```diff
@@ -1120,7 +1120,8 @@ class Shard extends EventEmitter {
                     oldMember = {
                         roles: member.roles,
                         nick: member.nick,
-                        premiumSince: member.premiumSince
+                        premiumSince: member.premiumSince,
+                        pending: member.pending
```
Comment (ghost) [inline lib/gateway/Shard.js:1124]: Would it be useful to have a `guildMemberVerified` (or similar) event here?

**Review by bsian03 — COMMENTED** [review]
JSDoc for the event will need to be updated as well

File: lib/gateway/Shard.js (hunk 1 above)
Comment (Khaaz) [reply lib/gateway/Shard.js:1124]: as mentionned in Discord, I don't think it's appropriate to have such event. Doesn't seem to fit in eris

**Comment by Dramex** [comment]
i don't know why this taking so long to accept

**Review by andreasclaesson — DISMISSED** [review]
This will be very useful, and everything should be working in the code.

**Comment by bsian03** [comment]
@Dramex Abal will merge the PR when he feels like doing so. If you really need this then just install the fork with `npm install Thewsomeguy/eris#dev --save`

File: lib/structures/Member.js [hunk 2]
```diff
@@ -27,6 +27,7 @@ const VoiceState = require("./VoiceState");
 * @prop {Number} joinedAt Timestamp of when the member joined the guild
 * @prop {String} mention A string that mentions the member
 * @prop {String?} nick The server nickname of the member
+* @prop {Boolean?} pending Whether the member has passed the guild's Membership Screening requirements
```
Comment (bsian03) [inline lib/structures/Member.js]: This should be marked as optional

**Review by Khaaz — APPROVED** [review]

**Review by Dramex — APPROVED** [review]

**Review by andreasclaesson — APPROVED** [review]

**Review by abalabahaha — APPROVED** [review]

**Pull request merged**